packages/api/cms-api/package.json

```diff
@@ -42,6 +42,7 @@
         "@nestjs/mapped-types": "^1.2.2",
         "@nestjs/passport": "^9.0.0",
         "@opentelemetry/api": "^1.4.0",
+        "@smithy/node-http-handler": "3.1.4",
```

Comment:
I hope that this ensures that v3.1.4 is always used. Though I'm not quite sure because in v6, `@aws-sdk/client-s3` was a peer dependency

Reply:
Doesn't matter after https://github.com/vivid-planet/comet/pull/2730/commits/b4b06b67a69230a25e1d93ad687826e63278b804